docusaurus/docs/React/release-guides/upgrade-to-v12.mdx

```diff
@@ -4,6 +4,40 @@ title: Upgrade to v12
 keywords: [migration guide, upgrade, v12, breaking changes]
 ---
 
+## Audio recordings transcoding
+
+Until now, the audio recordings were transcoded to `audio/mp3` format for inter-browser compatibility and size reduction. However, as of the v12, the MIME type `audio/wav` will be the default. The mp3 encoder used by the library is licensed under LGPL and integrators will have to opt-in to using the library in their apps.
```

Comment:
> The mp3 encoder used by the library is licensed under LGPL and integrators will have to opt-in to using the library in their apps.

This sentence creates uncertainty/fear/doubt. I think we should rephrase this to something else, or remove it. IMO, we are compliant with LGPL now. We make mp3 encoding opt-in.

Reply:
Updated